guides/doc-Administering_with_Ansible/master.adoc

```diff
@@ -3,7 +3,7 @@ include::common/header.adoc[]
 :configuring-ansible:
 :context: ansible-admin
 
-= Managing {Project} with Ansible
+= {AdministeringAnsibleDocTitle}
```

Comment:
Does the title of the guide still match the heading? If not, maybe this is a good opportunity to fix it, potentially to `doc-Administering_Project_with_Ansible`? If you make this change, then please on a separate commit which we then don't cherry-pick to 3.1 and 3.2.

Reply:
Yep, I missed this one, thanks!